packages/slate-react/src/components/slate.tsx

```diff
@@ -23,6 +23,7 @@ export const Slate = (props: {
   children: React.ReactNode
   onChange?: (value: Descendant[]) => void
 }) => {
+  // do we need to add onValueChange and onSelectorChange here?
```

Comment:
here is where I need to add the new handlers, @zbeyens?

and execute them inside [onContextChange](https://github.com/ianstormtaylor/slate/blob/main/packages/slate-react/src/components/slate.tsx?plain=1#L51C1-L61)?

or do you want me to add a new parameter on [onContextChange](https://github.com/ianstormtaylor/slate/blob/main/packages/slate-react/src/components/slate.tsx?plain=1#L51C1-L61), where we'll send the `onValueChange`/`onSelectorChange` (from [this line](https://github.com/ianstormtaylor/slate/pull/5523/files?plain=1#diff-7fe1360dfc4f25adbf2cb567cd2a6b3e655b3a62f07872dc072d64b5a80bcd82R374))